documentation/userguide/docs/datasets.md

```diff
@@ -257,6 +257,35 @@ Catalog and then click on Synchronize as we did in step 3.
 - Or directly, <a href="https://github.com/aws-samples/aws-glue-samples/tree/master/utilities/Hive_metastore_migration">migrating from Hive Metastore.</a>
 - there are more for sure :)
 
+#### (Going Further) Creating Filters on Tables
+
+Additionally, dataset owners can create column-level or row-level filters on their dataset tables to more granularly restrict data access when sharing with other teams. 
+
+To do so dataset owners can navigate to the **Filters** Tab for a given table and select **Add New Filter**:
+
+![dataset_table_filter](pictures/datasets/dataset_table_filter.png#zoom#shadow)
+
+When creating filters, you have the choice to create a column filter to filter by select colummns to include only OR you can create row level filters with one or more row expressions.
+
+An example of creating a column filter is below: 
+
+![dataset_table_filter_col](pictures/datasets/dataset_table_filter_col.png#zoom#shadow)
+
+This filter restricts access on the table to only the 3 selected columns: `book_id`, `author`, and `publisher`.
+
+An example of creating a row filter is below:
+
+![dataset_table_filter_row](pictures/datasets/dataset_table_filter_row.png#zoom#shadow)
+
+This filter restricts access to only rows where `book_id` is not null, `title` is LIKE `%Harry Potter%` AND `num_pages` is greater than `100`. It is important to note that:
+- The row filters that the intersection (logical 'AND') of the row expression(s) - if you need the union (logical 'OR') of multiple expressions you can create separate filters here and apply multiple to the table share item
+- When creating a new row expression be sure to save the row expression by clicking the save icon (highlighted in red in the above) before creating the filter
+
+Once the filters are created, they will show in the Filters Table Tab:
+
+![dataset_table_filter_filled](pictures/datasets/dataset_table_filter_filled.png#zoom#shadow)
+
```

Comment:
Maybe add a sentence on how to edit a filter and that they are editable

Reply:
They are not editable but will call that out here.

This is because editing a filter would lead to downstream complications with existing shares - maybe can look into editing metadata fields like name and subscription as an enhancement

Reply:
Updated